Packs/PAN-OS/Scripts/PanOSGetAvailablePANOSSoftware/PanOSGetAvailablePANOSSoftware.yml

```
@@ -0,0 +1,69 @@
+args:
+- description: The string by which to filter the results to only show specific hostnames or serial numbers.
+  name: device_filter_string
+  type: unknown
+- description: The target number of the firewall. Used only on a Panorama instance.
+  name: target
+- description: The Instance Name of the PAN-OS Integration to use. Specify only one Instance.
+  name: panos_instance_name
+- auto: PREDEFINED
+  defaultValue: "yes"
+  description: Whether to return only images newer than currently installed.
+  name: newer_images_only
+  predefined:
+  - "yes"
+  - "no"
+comment: This script is a wrapper for the 'pano-os-platform-get-available-software' command, adding options to return only images newer than is currently installed to minimize amount of context data generated.
+commonfields:
+  id: PAN-OS-GetAvailablePANOSSoftware
+  version: -1
+dependson:
+  must:
+  - Panorama|||pan-os-platform-get-available-software
+dockerimage: demisto/python3:3.12.11.4095827
+enabled: true
+engineinfo: {}
+name: PAN-OS-GetAvailablePANOSSoftware
+outputs:
+- contextPath: PANOS.SoftwareVersions.Summary.hostid
+  description: The host ID.
+  type: String
+- contextPath: PANOS.SoftwareVersions.Summary.version
+  description: The software version in Major.Minor.Maint format.
+  type: String
+- contextPath: PANOS.SoftwareVersions.Summary.filename
+  description: The software version filename.
+  type: String
+- contextPath: PANOS.SoftwareVersions.Summary.size
+  description: The size of the software in MB.
+  type: String
+- contextPath: PANOS.SoftwareVersions.Summary.size_kb
+  description: The size of the software in KB.
+  type: String
+- contextPath: PANOS.SoftwareVersions.Summary.release_notes
+  description: The link to version release notes on PAN knowledge base.
```

Should we spell out PAN --- and can we add a link? 
```suggestion
  description: The link to version release notes in the PAN knowledge base.
```